Source/Core/TaskProcessor.js

```diff
@@ -352,7 +351,6 @@ TaskProcessor.prototype.destroy = function () {
   if (defined(this._worker)) {
     this._worker.terminate();
   }
-  return destroyObject(this);
```

Comment:
Thanks for the PR @H91011! This change is a fairly sweeping one, which in turn is causing a lot of the unit tests to fail. Would you mind explaining in more detail why the task processor is not getting properly destroyed? We'll want to make sure this solution is generic to all uses of the `TaskProcessor`, not just its use in `Primitive.js`.

Reply:
Yes I realize that because of unit tests. I try to some changes but I can not pass.
 you can take a look this topic I share results: https://community.cesium.com/t/memory-usage-did-not-reduce-after-remove-datasource/15203/4
```
For instance, if there’s an object in a global variable, and that object has a property referencing another object, that object is considered reachable. And those that it references are also reachable. 
```
you can read here : https://javascript.info/garbage-collection

Your task processor object properties is still reachable after your progress so they can not release from memory. I found this way for release memory. The code is a bit crowded here so I can found this after search one week. 

I think there is a misunderstanding about cesium destroy mechanism, it says destroy but it actually not destroy complete. I think if you say destroy it must be destroy real, not only remove functions. 

There is another leaks for destroying objects, after destroy function some gl objects are still stay reachable so it cause memory leak also(I realize in points object)

Reply:
Based on your [example](https://github.com/CesiumGS/cesium/pull/9984/files#diff-a2fe521025654f48a319ee31aead9312fbf0ef4537cf188b488701d6868430bb), I'm not sure this is actually a memory leak. The worker associated with task processor is intentionally not destroyed after creating the geometry because it must remain around to do other work. This is why the unit tests are failing in this branch: They are trying to update the primitive, but can't since the resources have been destroyed. 

If you're interested in getting better performance while your app is idle, I would just [request render mode](https://cesium.com/blog/2018/01/24/cesium-scene-rendering-performance/). If you're still seeing problems, I would ask that you take this to the [Cesium Forum](https://community.cesium.com/) where there are members of the community and developers from the team who can help troubleshoot.

I'm going to close this PR for now.

Reply:
then why its not release memory ?  in my topic at forum 
I share result after add 1k line (each line has 1k vetex) it still hold 4GB memory is this normal ? I dont think this amount of objects hold 4GB memory. And when you delete it it still hold 4 GB memory. if you want to explain  I wait here  https://meet.google.com/han-wszx-erd

Reply:
See these explanations on the forum of [why immediately destroying a primitive does not immediately show up in the profiler](https://community.cesium.com/t/why-gpu-memory-no-release/16866) and [why we do not destroy web workers agressively](https://community.cesium.com/t/webworkers-memory-issues/11683).

Reply:
ok nevermind, I work on project like this about 3 years and our application does not have leak like this. It is a leak. If you dont want its ok.